### PR TITLE
feat(template): Add `PhysicalLink` link

### PR DESCRIPTION
**Pull request by ewuerger**

### CODE REVIEW

File: template/.polarion/hats/_default/tracker/form-layout.xml [hunk 1]
```diff
@@ -13,12 +13,15 @@ SPDX-License-Identifier: Apache-2.0 -->
...
     <field id="description" />
-
-    <panel description="Custom Fields">
-        <field
-            id="@allCustomFields,-owner,-sortOrder,-manualQuality,-qualitySuspect,-automatics,-language,-capellaId" />
+    <panel description="Generic Capella Fields">
+        <field id="uuid_capella,parent,parent_reverse" readOnly="true" />
+    </panel>
+    <panel description="Capella Link Fields">
+        <field id="preCondition,postCondition,allocated_functions,allocated_functions_reverse,allocated_functional_exchanges,allocated_functional_exchanges_reverse,exchange_items,exchange_items_reverse,involved_activities,involved_activities_reverse,involved_entities,involved_entities_reverse,diagram_elements,diagram_elements_reverse,state_machines,state_machines_reverse,constraints,constraints_reverse,output_exchanges,output_exchanges_reverse,input_exchanges,input_exchanges_reverse,activities,activities_reverse,outputs,outputs_reverse,inputs,inputs_reverse,description_reference,description_reference_reverse,scenarios,scenarios_reverse,groups,groups_reverse,physical_links,physical_links_reverse,exchanges,exchanges_reverse" readOnly="true" />
```
Comment (micha91) [inline template/.polarion/hats/_default/tracker/form-layout.xml:20]: preCondition and postCondition are no links, are they?

File: template/.polarion/tracker/fields/workitem-link-role-enum.xml [hunk 2]
```diff
@@ -2,242 +2,159 @@
...
+    <option id="diagram_elements" description="Linking to elements viewable in a diagram." name="shows" oppositeName="is shown in">
+        <rule from="diagram" />
+    </option>
+    <option id="realized_functions" description="A Function realizes a lower-layer Function." name="realized" oppositeName="realized in">
+        <rule from="physicalFunction" to="logicalFunction" />
+        <rule from="logicalFunction" to="systemFunction" />
+        <rule from="systemFunction" to="operationalActivity" />
+    </option>
+    <option id="physical_links" name="physical links" oppositeName="physical link of" sortOrder="118">
+        <rule from="componentExchange, physicalComponentNode, physicalActor" to="physicalLink"/>
```
Comment (micha91) [inline template/.polarion/tracker/fields/workitem-link-role-enum.xml:158]: I think we miss the `exchange` role here, aren't we?

**Review by micha91 — CHANGES_REQUESTED** [review]

File: template/.polarion/tracker/fields/workitem-link-role-enum.xml (hunk 2 above)
Comment (ewuerger) [reply template/.polarion/tracker/fields/workitem-link-role-enum.xml:158]: There is `exchanges` with the name exchange.

File: template/.polarion/tracker/fields/workitem-link-role-enum.xml [hunk 3]
```diff
@@ -2,242 +2,159 @@
...
+        <rule from="operationalEntity" to="operationalActivity" />
+    </option>
+    <option id="inputs" description="An OperationalActivity has incoming OperationalInteractions which transport input data/exchange items." name="incoming" oppositeName="is input on" sortOrder="111">
+        <rule from="operationalActivity" to="operationalInteraction" />
+    </option>
+    <option id="outputs" description="An OperationalActivity has outgoing OperationalInteractions which transport output data/exchange items." name="outgoing" oppositeName="is output on" sortOrder="112">
+        <rule from="operationalActivity" to="operationalInteraction" />
+    </option>
+    <option id="exchanges" description="The exchanges connecting the Function." name="exchange" oppositeName="is exchange on" sortOrder="110">
+        <rule from="systemFunction, logicalFunction, physicalFunction" to="functionalExchange" />
```
Comment (micha91) [inline template/.polarion/tracker/fields/workitem-link-role-enum.xml:132]: We are missing a rule for component exchanges here

**Review by micha91 — CHANGES_REQUESTED** [review]

**Pull request merged**